From 1d86ce7e6ea9fae8285eed23d838f87ab3ac083c Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Tue, 19 Aug 2025 23:28:35 +0200
Subject: [PATCH] src/{svgtiny_gradient,svgtiny_list}.c: add missing <stdlib.h>

Needed to get the prototype for malloc()/free().

Upstream: N/A, already fixed upstream in a later release (but as part
of a larger commit)
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 src/svgtiny_gradient.c | 1 +
 src/svgtiny_list.c     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/svgtiny_gradient.c b/src/svgtiny_gradient.c
index 3a8db73..5172223 100644
--- a/src/svgtiny_gradient.c
+++ b/src/svgtiny_gradient.c
@@ -8,6 +8,7 @@
 #include <assert.h>
 #include <math.h>
 #include <string.h>
+#include <stdlib.h>
 #include "svgtiny.h"
 #include "svgtiny_internal.h"
 
diff --git a/src/svgtiny_list.c b/src/svgtiny_list.c
index 53cfb34..91e2c98 100644
--- a/src/svgtiny_list.c
+++ b/src/svgtiny_list.c
@@ -11,6 +11,7 @@
  */
 
 #include <assert.h>
+#include <stdlib.h>
 #include "svgtiny.h"
 #include "svgtiny_internal.h"
 
-- 
2.50.1

